packages/project-disaster-trail/src/components/Game/Orb.js

```diff
@@ -1,37 +1,46 @@
 /** @jsx jsx */
 import { Component } from "react";
-import PropTypes from "prop-types";
 import { jsx, css } from "@emotion/core";
+import RadialGauge from "./RadialGauge";
 
-const defaultStyle = css`
-  height: 30px;
-  width: 30px;
-  border-radius: 15px;
-  background-color: blue;
+const orbContainerStyle = css`
+  position: relative;
+`;
+
+const circleDefaultStyle = css`
+  transition: transform 1s;
   position: absolute;
+  top: 10px;
+  left: 10px;
+  z-index: 10;
+  background-color: mediumSeaGreen;
+  -webkit-transition: background-color 1000ms linear;
+  -ms-transition: background-color 1000ms linear;
+  transition: background-color 1000ms linear;
   cursor: pointer;
   opacity: 0.8;
+
   &:hover {
     opacity: 1;
   }
-`;
 
-const pressStyle = css`
-  background-color: #000;
-  -webkit-transition: background-color 1000ms linear;
-  -ms-transition: background-color 1000ms linear;
-  transition: background-color 1000ms linear;
+  &.circle-press-style {
+    background-color: mediumAquamarine;
+    -webkit-transition: background-color 1000ms linear;
+    -ms-transition: background-color 1000ms linear;
+    transition: background-color 1000ms linear;
+  }
 `;
 
 const defaultState = {
   durationRequired: 1000,
+  orbSize: 60,
```

Comment:
Woohoo!

Reply:
Ha! Just making it arbitrarily a bit bigger for ease of development ;)